client/src/ui/css/App.css

```diff
@@ -24,7 +24,7 @@
   margin: 120px 0px 0px 120px;
 }
 
-.background-gradient{
+.background-gradient_summer{
```

Comment:
Does this css get used? Looking at the div we have `background-gradient_" + dayclass + monthclass` and from the looks of it dayclass is never the empty string so it does not seem like this is used?

Reply:
It doesn't get used, I'll take it out